helix-core/src/test/java/org/apache/helix/integration/task/TestAddDeleteTask.java

```diff
@@ -89,9 +100,16 @@ public void testAddJobMissing() throws Exception {
     } catch (IllegalArgumentException e) {
       // Helix Exception is expected because job config is missing
     }
+
+    try {
+      _driver.deleteTask(workflowName, jobName, task.getId());
+      Assert.fail("Exception is expected because job config is missing");
+    } catch (IllegalArgumentException e) {
+      // Helix Exception is expected because job config is missing
```

Comment:
Helix exception? While illegal argument exception is caught.

Reply:
Good catch. Let me fix the comments.